### ISSUE TITLE
install.sh failed + ImagePullBackError

### ISSUE THREAD

**Issue by veromos** [Dynatrace/dynatrace-operator#422]
Good morning,

I tried to install Dynatrace on EKS, it works fine few weeks ago, and i had to redo the install, but now i encountered this error with v0.2.2:

```
Applying DynaKube CustomResource...
Error from server (InternalError): error when creating "STDIN": Internal error occurred: conversion webhook for dynatrace.com/v1alpha1, Kind=DynaKube failed: Post "https://dynatrace-webhook.dynatrace.svc:443/convert?timeout=30s": no endpoints available for service "dynatrace-webhook"
```

So I download the latest install.sh release with wget, the went fine.

`Kubernetes monitoring successfully setup.  `

But on my Kubernetes cluster, i encounter these errors ImagePullBackOff :

```
Failed to pull image "MY_API_URL/linux/oneagent:latest": rpc error: code = Unknown desc = Error response from daemon: Get https://MY_API_URL_SHORT/v2/: x509: certificate signed by unknown authority
``` 
![image](https://user-images.githubusercontent.com/15998575/147126318-4e178676-c43c-4fe1-bb5f-ddb6718cd998.png)

And why is the operator is installing dynakube-oneagent instead of dynakube-classic ?

**Comment by luhi-DT** [comment]
Hi,
did you properly invoke the install.sh with the right parameters? Just applying the install.sh without setting api-url, paas-token, api-token and cluster name does not work. The whole command is provided in the Dynatrace Web UI under Manage -> Deploy Dynatrace -> Kubernetes. [Official Docu for automated Setup](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/set-up-k8s-monitoring#:~:text=a%20PaaS%20token.-,Automated,-In%20the%20Dynatrace)

**Comment by veromos** [comment]
Hi,

Yes I used that command with valid and generated passtoken, apitoken, apiurl, clustername and I also add the option to skip SSL Cert check so I don't know why it's telling me my certificate is signed by unknown authority.

**Comment by luhi-DT** [comment]
Hi, this looks like a very specific problem and not a general, therefore i would suggest to open a ticket on our support page -> [Dynatrace Help](https://www.dynatrace.com/support/help)

**Comment by veromos** [comment]
Small update:

the actual errors are : 

```
dynakube-oneagent error :
Failed to pull image "dynatrace.si2m.tec/e/ede30eba-610e-4c16-85f5-5abece535fc3/linux/oneagent:latest": rpc error: code = Unknown desc = Error response from daemon: Get "https://dynatrace.my-entreprise.com/v2/": x509: certificate signed by unknown authority
```

```
dynakube-activegate error :
Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "240704fdb13097e30f717e95a5f509a5c529670d5873abf118e0f366d18509e9" network for pod "dynakube-activegate-0": networkPlugin cni failed to set up pod "dynakube-activegate-0_dynatrace" network: add cmd: failed to assign an IP address to container
```

It seems like the --skip-ssl-verification option is not working anymore ? (worked in release 2.2)

**Comment by JKrehling** [comment]
When you get an imagepullerror from the describe its the cacerts on the host machine.  You can't change that from any yaml file.  
Failed to pull image "dynatrace.si2m.tec/e/ede30eba-610e-4c16-85f5-5abece535fc3/linux/oneagent:latest": rpc error: code = Unknown desc = Error response from daemon: Get "https://dynatrace.my-entreprise.com/v2/": x509: certificate signed by unknown authority

It's not from anything you can change in the yaml.  Whatever certificate is in dynatrace.my-entreprise.com is not in the /etc/pki/certs store (or wherever certs are stored on your OS).  The image in 0,3.0 is pulled from the cluster, not dockerhub like it was in 0.2.2.

**Comment by veromos** [comment]
Yes exactly, we figured out that installling the Certificate Authority on our node groupe resolved the problem.
Thanks.
